web-common/src/features/dashboards/url-state/getExploreStateFromSessionStorage.ts

```diff
@@ -0,0 +1,77 @@
+import { convertPresetToExploreState } from "@rilldata/web-common/features/dashboards/url-state/convertPresetToExploreState";
+import { getExplorePresetForWebView } from "@rilldata/web-common/features/dashboards/url-state/explore-web-view-store";
+import { FromURLParamViewMap } from "@rilldata/web-common/features/dashboards/url-state/mappers";
+import { ExploreStateURLParams } from "@rilldata/web-common/features/dashboards/url-state/url-params";
+import {
+  type V1ExplorePreset,
+  type V1ExploreSpec,
+  V1ExploreWebView,
+  type V1MetricsViewSpec,
+} from "@rilldata/web-common/runtime-client";
+
+/**
+ * Redirects to a view with params loaded from session storage.
+ * 1. If no param is set then load the params for the default view from session storage.
+ * 2. If only view param is set then load the params from session storage.
+ * 3. If view=ttd and `measure` is the only other param set load from session storage.
+ */
+export function getExploreStateFromSessionStorage(
```

Comment:
I'd expect to see a call to session storage in this function. Perhaps `getExplorePresetForWebView` (where I see the call to session storage actually happens) is an unnecessary additional layer.